pkg/networkservice/common/excludedprefixes/server.go

```diff
@@ -44,26 +47,35 @@ type excludedPrefixesServer struct {
 
 func (eps *excludedPrefixesServer) init() {
 	logger := trace.Log(eps.ctx)
-	updatePrefixes := func(bytes []byte) {
+	onUpdate := func(_ fsnotify.Event) error {
+		bytes, err := ioutil.ReadFile(filepath.Clean(eps.configPath))
```

Comment:
Can we here extract path from the event?

Reply:
Yes, we can get `event.Name` and it should be the path for the file triggered the event.